apps/web/scripts/installExternalCourses.py

```diff
@@ -0,0 +1,20 @@
+import json
+import sys
+import os
+
+def main():
+    if len(sys.argv) != 2:
+        exit(f"Usage: {sys.argv[0]} PATH_TO_CONFIG_FILE")
+    filename = sys.argv[1]
+    with open(filename) as fh:
+        config = json.load(fh)
+    for course in config:
```

Comment:
I would skip an empty line here. I wonder if there's a way to configure our linter tools to enforce this :thinking: